### ISSUE TITLE
Allow specifying full namespace as a `--namespace` value

### ISSUE THREAD

**Issue by rabraghib** [zenstruck/foundry#169]
it would be good if we are able to specifying the full namespace (`App\..` or `App\Tests\...`) since Its standard for namespaces and its always the first way anyone will try to use the `--namespace` flag with
& also if we can set its default value in the config
```yml
zenstruck_foundry:
    default:
        namespace: App\Factory
```

**Comment by kbond** [comment]
Ok, I think I see what you mean about the full namespace: `bin/console make:factory --namespace=App\\My\\Factory` generates a factory with the namespace `App\App\My\Factory`. Do I understand correctly? It looks like I can fix this.

For the default namespace config: I'd like to avoid this and advise using the [maker-bundle's](https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html#configuration) option for this. Down the road we might actually move this maker into the maker-bundle itself. I'd like to avoid adding maker-specific config to this library so we can make that transition as seamless as possible.

**Comment by rabraghib** [comment]
yes It is as you said.
Thank you for the swift response!

**Comment by kbond** [comment]
Oops, sorry @rabraghib, didn't realize you were working on a fix.

**Comment by rabraghib** [comment]
@kbond lol its just a simple function

**Comment by kbond** [comment]
Released in [v1.11.1](https://github.com/zenstruck/foundry/releases/tag/v1.11.1).
